### ISSUE TITLE
GitHub Latest button broken

### ISSUE THREAD

**Issue by jerone** [Ede123/userscripts#1]
With the latest changes from GitHub the button from GitHub Latest looks weird:
![2016-02-12 19_46_50-new issue ede123_userscripts](https://cloud.githubusercontent.com/assets/55841/13016831/642030da-d1c1-11e5-8a33-05a4df8b5b05.png)

**Comment by Ede123** [comment]
Thanks for the report!

Should be fixed by 76a7f830b63f0d0eb6ace87f08bb356cf69457dc.

**Comment by jerone** [comment]
Confirmed working again. Thanks!

**Comment by jerone** [comment]
It looks like GitHub did some more adjustments to their site; the button won't show up at now.

The following error message appeared in the Browser Console:

```log
button.firstElementChild.firstChild.setAttribute is not a function
```

**Comment by Ede123** [comment]
Fixed in c2ca2451ea7f0123a6afeb5a51b5f291278fb219, be sure to let me know when they break it again :wink:

Sadly GitHub doesn't give nice IDs to their elements that I could select reliably, so I could only try to match random attributes (which are just as likely to change, tough).

**Comment by jerone** [comment]
I [understand](https://github.com/jerone/UserScripts#github) your problem. Thanks for the useful script!
